python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -1057,6 +1083,23 @@ def do_evaluate(
                 )
                 arg = evaluated.obj_scalar if evaluated.is_scalar else evaluated.obj
             return Column(plc.replace.replace_nulls(column.obj, arg))
+        elif self.name == "pow":
+            column = self.children[0].evaluate(df, context=context, mapping=mapping)
+
+            evaluated = self.children[1].evaluate(df, context=context, mapping=mapping)
```

Comment:
`(column, power) = (child.evaluate(df, context=context, mapping=mapping) for child in self.children)`


And then we notice if children is ever too long.